src/sentry/issues/occurrence_consumer.py

```diff
@@ -261,11 +261,13 @@ def _process_message(
         sampled=True,
     ) as txn:
         try:
-            if message.get("payload_type") == PayloadType.OCCURRENCE.value:
-                return process_occurrence_message(message, txn)
-            elif message.get("payload_type") == PayloadType.STATUS_CHANGE.value:
+            # Assume messaged without a payload type are of type OCCURRENCE
+            payload_type = message.get("payload_type", PayloadType.OCCURRENCE.value)
+            if payload_type == PayloadType.STATUS_CHANGE.value:
                 group = process_status_change_message(message, txn)
                 return None, GroupInfo(group=group, is_new=False, is_regression=False)
+            elif payload_type == PayloadType.OCCURRENCE.value:
+                return process_occurrence_message(message, txn)
```

Comment:
We need an `else` here that logs a stat if the payload type doesn't match anything we expect, so that we can have alerts in datadog if things go wrong.